Sources/Resources/Media.xcassets/forageLogo.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "forage_logo.pdf",
```

Comment:
why was this a PDF before?

Reply:
I'm going to toss that up to a mistake :)